### ISSUE TITLE
can not pass the test 

### ISSUE THREAD

**Issue by haobo724** [princeton-vl/lietorch#6]
Hi ,

At first, Thank you so much  providing this library! 

I got some trouble when I first used it. I followed the intro step install the library successfully But when I run test, in the section about GPU Test, the program gets stuck, like in an infinite loop. After a few minutes the script will end by itself without error. In console shows `Testing lietorch forward pass (GPU) ...

Process finished with exit code -1073741819 (0xC0000005)` 

Obviously this is not a normal exit, there may be a memory leak or something else. 

And when I run the demo.py of Raft3d project , the same issue occured again, the data could transfer into the model but model can not  complete a complete calculation. I noticed it will get stuck in projective_ops.py in line  42: `  X1 = Ts * X0` .

My GPU is GTX 1070 , run on win 10 pytorch ==1.8 cudatookit =11.1 python 3.8

**Comment by yclicc** [comment]
I'm still not sure _why_ it should be, but I managed to fix this on Windows by making all the CUDA kernels in lietorch_gpu.cu accept regular pointers where currently they accept const pointers. Hopefully someone who knows more about CUDA and Windows can make a proper pull request.

**Comment by iampalop** [comment]
Have you solved it yet? can not pass the test neither

**Comment by yclicc** [comment]
> Have you solved it yet? can not pass the test neither

Yep, see my comment above for a solution, though I don't think it's been put into main yet. 

See also the discussion at https://github.com/princeton-vl/lietorch/issues/22#issuecomment-1921513151

**Comment by iampalop** [comment]
thank you so much!
